test/meter/meter-linear.visual-diff.js

```diff
@@ -0,0 +1,35 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('visual-diff');
```

Comment:
We've moved the visual-diff lib (and published to NPM) so this line will need to be updated to partly fix the PR build:
```
const VisualDiff = require('@brightspace-ui/visual-diff');
```